contrib/actions/ovh-publish-metrics

```diff
@@ -0,0 +1,78 @@
+
+name = "OVH_PublishMetric"
+description = "Publish a metric"
+
+// Requirements
+requirements = {
+  "curl" = {
+		type = "binary"
+		value = "curl"
+	}
+	"bash" = {
+		type = "binary"
+		value = "bash"
+	}
+}
+
+// Parameters
+parameters = {
+	 "name" = {
+		type = "string"
+		description = "Name of you metric (optional)"
+		value = "cds"
+	}
+	 "labels" = {
+		type = "text"
+		description = "Labels of your metric (one K/V per line separated by a space)"
+		value = "app {{.cds.application}}
+env {{.cds.environment}}"
+	}
+	 "value" = {
+		type = "string"
+		description = "Value of your metric"
+		value = "T"
```

Comment:
as a user I don't understand what means T. Maybe an example in description or better description ?

Reply:
or a link on metrics website on action description?